CrazyCanvas/Saved_Servers.json

```diff
@@ -0,0 +1,6 @@
+{
+    "SERVER_IP": [
+        "bajs",
+        "HejSimon"
+    ]
```

Comment:
Maybe the test name is not user friendly
This is work to save and load an address?
Does it care about the port?